sample_apps/rust/README.md

```diff
@@ -0,0 +1,70 @@
+# Rust Sample Application
+
+This sample application shows how to:
+1. Create a table enabled with magnetic tier upsert.
+2. Ingest data with multi measure records.
+3. Execute queries and interpret their results.
+
+This application populates a table with ~63K rows of sample multi-measure-value data (provided as csv data), and runs sample queries to jumpstart your evaluation and/or proof-of-concept applications with Amazon Timestream for LiveAnalytics.
+
+## Sample Application Usage
+
+### Prerequisites
+
+1. Install [Rust](https://www.rust-lang.org/tools/install).
+
+2. Create a Timestream for LiveAnalytics database called `devops_multi_sample_application` with a table named `host_metrics_sample_application`.
```

Comment:
Not sure if I should remove this as the `ingestion_csv_sample` application should create the necessary database and table automatically.

Reply:
Yes let's remove it, I added the dynamic DB and table creation.

Reply:
I have removed this step and replaced it with setting up [SDK authentication with AWS](https://docs.aws.amazon.com/sdk-for-rust/latest/dg/credentials.html). I changed the description of the database and table to say they get created if they don't already exist.